tests/test_files.py

```diff
@@ -1,52 +1,52 @@
 import os
 import unittest
-
+from PIL import Image as PillowImage
 import simpleocr.files
-from simpleocr.files import open_img
+from simpleocr.files import open_image
 
 TEST_FILE = 'digits1'
 TEST_FILE_EXT = 'digits1.png'
 UNICODE_TEST_FILE = 'unicode1'
 
+
 class TestImageFile(unittest.TestCase):
-    def test_open_img(self):
+    def test_image_from_file(self):
```

Comment:
Generally you name tests after the functions they test. You're not testing `from_file` here